addons/mediarecorder/userscript.js

```diff
@@ -10,6 +10,18 @@ export default async ({ addon, console, msg }) => {
   let recordBuffer = [];
   let recorder;
   let timeout;
+
+  const mimeType = [
+    // Chrome and Firefox only support encoding as webm
+    // VP9 is preferred as its playback is better supported across platforms
+    "video/webm; codecs=vp9",
+    // Firefox only supports encoding VP8
+    "video/webm",
+    // Safari only supports encoding H264 as mp4
+    "video/mp4",
```

Comment:
As of Chrome 126, `video/mp4` is supported: https://chromestatus.com/feature/5163469011943424

@GarboMuffin Should we move it to the top of this list?

Reply:
i think that would make a lot of people happier